supertokens_python/recipe/session/with_jwt/recipe_implementation.py

```diff
@@ -13,9 +13,12 @@
 # under the License.
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, Dict, Union
```

Comment:
Missing changes for session_class inside with_jwt? https://github.com/supertokens/supertokens-node/pull/278/files#diff-c2df0b9555fe0bb05c17bc4a8ae33caf9c9f9f02a8d2895383d1f0c74fe260e4R59